@xen-orchestra/lite/src/views/vm/system/VmGeneralInfo.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <UiCard class="vm-general-info">
```

Comment:
Following discussions on MM with @J0ris-K and @ByScripts, we concluded that a style-less class solely for debugging purposes lacks utility and can be removed, along with its counterparts in other components (`VmGraphics`, `VmManagement`, etc.)